Sources/ContainerClient/Utility.swift

```diff
@@ -185,8 +185,38 @@ public struct Utility {
                         volume = try await ClientVolume.inspect(parsed.name)
                     }
                 } else {
-                    // Named volume so it must already exist
-                    volume = try await ClientVolume.inspect(parsed.name)
+                    // Named volume - create if doesn't exist, inspect if it does
+                    var volumeCreated = false
```

Comment:
This is the smallest of nits, but could you change this to minimize the usage of mutable state?

Using `let volumeCreated: Bool` here helps avoid subtle bugs, since you need to explicitly indicate how things are handled for each conditional block.

It's typically harder for both humans and machines to reason about code that uses `var`.